sdk/react/languageSelectionPreview.js

```diff
@@ -18,6 +18,7 @@ var {
 var LanguageSelectionPreview = React.createClass({
   propTypes: {
     config: React.PropTypes.object,
+    selectedLanguage: React.PropTypes.string,
```

Comment:
I talked with Steve he is said to fix android now and open ticket for iOS for future.